src/NexusMods.App.UI/Controls/MarkdownRenderer/MarkdownRendererView.axaml

```diff
@@ -11,51 +11,110 @@
-                <Setter Property="Foreground" Value="{StaticResource NeutralModerateBrush}" />
+                <Setter Property="FontWeight" Value="SemiBold" />
+                <Setter Property="FontSize" Value="24" />
+                <Setter Property="LineHeight" Value="30" />
+                <Setter Property="Foreground" Value="{StaticResource NeutralStrongBrush}" />
             </Style>
 
             <Style Selector="ctxt|CTextBlock.Heading3">
                 <Setter Property="FontFamily" Value="{StaticResource FontHeadlinesSemiBold}" />
-                <Setter Property="Foreground" Value="{StaticResource NeutralModerateBrush}" />
+                <Setter Property="FontWeight" Value="SemiBold" />
+                <Setter Property="FontSize" Value="18" />
+                <Setter Property="LineHeight" Value="24.75" />
+                <Setter Property="Foreground" Value="{StaticResource NeutralStrongBrush}" />
             </Style>
 
             <Style Selector="ctxt|CTextBlock.Heading4">
-                <Setter Property="FontFamily" Value="{StaticResource FontHeadlinesSemiBold}" />
-                <Setter Property="Foreground" Value="{StaticResource NeutralModerateBrush}" />
+                <Setter Property="FontFamily" Value="{StaticResource FontBodyBold}" />
+                <Setter Property="FontWeight" Value="Bold" />
+                <Setter Property="FontSize" Value="16" />
+                <Setter Property="LineHeight" Value="24" />
+                <Setter Property="Foreground" Value="{StaticResource NeutralStrongBrush}" />
             </Style>
 
             <Style Selector="ctxt|CTextBlock.Heading5">
                 <Setter Property="FontFamily" Value="{StaticResource FontHeadlinesSemiBold}" />
-                <Setter Property="Foreground" Value="{StaticResource NeutralModerateBrush}" />
+                <Setter Property="FontWeight" Value="SemiBold" />
+                <Setter Property="FontSize" Value="12" />
+                <Setter Property="LineHeight" Value="16.5" />
+                <Setter Property="Foreground" Value="{StaticResource NeutralStrongBrush}" />
             </Style>
 
             <Style Selector="ctxt|CTextBlock.Heading6">
                 <Setter Property="FontFamily" Value="{StaticResource FontHeadlinesSemiBold}" />
+                <Setter Property="FontWeight" Value="SemiBold" />
+                <Setter Property="FontSize" Value="10" />
+                <Setter Property="LineHeight" Value="15" />
+                <Setter Property="Foreground" Value="{StaticResource NeutralStrongBrush}" />
+            </Style>
+
+            <Style Selector="ctxt|CTextBlock.Paragraph">
+                <Setter Property="FontFamily" Value="{StaticResource FontBodyRegular}" />
+                <Setter Property="FontWeight" Value="Normal" />
+                <Setter Property="FontSize" Value="14" />
+                <!-- <Setter Property="LineHeight" Value="21" /> -->
```

Comment:
Add a comment explaining why LineHeight is problematic, so we avoid trying to set it again in the future.